src/cast_core/src/actors/RunCommand.cpp

```diff
@@ -28,11 +26,76 @@
 
 #include <boost/log/trivial.hpp>
 
+#include <cast_core/actors/RunCommand.hpp>
+
 #include <gennylib/ExecutionStrategy.hpp>
 #include <gennylib/MongoException.hpp>
 #include <gennylib/v1/RateLimiter.hpp>
 #include <gennylib/value_generators.hpp>
 
+namespace {
+
+/**
+ * @private
+ * @param command passed to run_command()
+ * @return if the command requires waiting for a stepdown/election to complete before continuing.s
+ */
+bool requiresWaitForStepdown(bsoncxx::document::view& command) {
+    return command.find("replSetStepDown") != command.end();
```

Comment:
I'm pretty nervous about arbitrarily changing behavior based on the underlying command. That's a good way to make users never trust your config language again. If this *has* to be bundled into the existing RunCommand actor instead of a specialization of it, would you please make it activated via an additional flag in the operation?

(I can kinda see the point of inline to RunCommand, but it brings me back to my angst about how it makes more sense to me for each phase to potentially be a different actor that may share state with actors on previous phases.)

Reply:
Putting a bit more thought at this, it's perfectly fine to generate a warning that `replSetStepDown` without that flag will behave in a weird way. We can even do that in the config stage.

Reply:
This was at my request. Ryan originally proposed exposing a boolean flag on the RunCommand actor to control this behavior, but it seemed to me like something we'd eventually remove in favor of having this behavior managed by a new (but currently nonexistent) Nemesis actor.